targets/targets.json

```diff
@@ -1134,6 +1134,33 @@
             }
         }
     },
+    "MTB_ADV_WISE_1530": {
+        "inherits": ["FAMILY_STM32"],
+        "core": "Cortex-M4F",
+        "extra_labels_add": ["STM32F4", "STM32F412xG", "STM32F412RG", "CORDIO"],
+        "device_has_add": ["CAN", "LOWPOWERTIMER", "SERIAL_ASYNCH", "SERIAL_FC", "TRNG", "FLASH"],
+        "config": {
+            "clock_source": {
+                "help": "Mask value : USE_PLL_HSE_EXTC | USE_PLL_HSE_XTAL (need HW patch) | USE_PLL_HSI",
+                "value": "USE_PLL_HSE_EXTC|USE_PLL_HSE_XTAL|USE_PLL_HSI",
+                "macro_name": "CLOCK_SOURCE"
+            }
+        },
+        "overrides": {
+            "stdio_uart_tx": "PB_10",
+            "stdio_uart_rx": "PC_11"
+        },
+        "device_has_add": ["LOWPOWERTIMER", "SERIAL_ASYNCH", "SERIAL_FC"],
+        "release_versions": ["5"],
+        "device_name": "STM32F412RG"
+    },
+    "MTB_USI_WM_BN_BM_22": {
+        "inherits": ["MTB_ADV_WISE_1530"],
```

Comment:
Derivative target must not inherit from another target. It should inherit from parent MCU..

Reply:
Will remove usi from here. There will be a lot of duplicate definitions, since the wise-1530 and USI targets are basicly the same targets. According the hw specification the wise-1530 includes usi chip. Only the tx&rx pins are diffrent with these 2 targets